From deb03351fe4ad90db127fbd9fe77473cbb7efe3b Mon Sep 17 00:00:00 2001
From: Shunsuke Kataoka <shunsuke.kataoka.df@renesas.com>
Date: Fri, 26 Jul 2013 17:45:57 +0900
Subject: [PATCH 0759/1083] ARM: shmobile: r8a7790: Remove dma channel clear
 offset

Since chclr_offset member of sh_dmadesc_channel structure is not used
in chclr_write, the third setup of DMA_CHANNEL was changed to 0.

Signed-off-by: Shunsuke Kataoka <shunsuke.kataoka.df@renesas.com>
---
 arch/arm/mach-shmobile/setup-r8a7790.c | 56 +++++++++++++++++-----------------
 1 file changed, 28 insertions(+), 28 deletions(-)

diff --git a/arch/arm/mach-shmobile/setup-r8a7790.c b/arch/arm/mach-shmobile/setup-r8a7790.c
index 8ab8147..e2dbeea 100644
--- a/arch/arm/mach-shmobile/setup-r8a7790.c
+++ b/arch/arm/mach-shmobile/setup-r8a7790.c
@@ -1160,19 +1160,19 @@ static const struct sh_audmapp_slave_config r8a7790_audmapp_slaves[] = {
 };
 
 static const struct sh_dmadesc_channel r8a7790_audma_channels[] = {
-	DMA_CHANNEL(0x00008000, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008080, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008100, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008180, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008200, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008280, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008300, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008380, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008400, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008480, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008500, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008580, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008600, 0x40, 0x00000080),
+	DMA_CHANNEL(0x00008000, 0x40, 0),
+	DMA_CHANNEL(0x00008080, 0x40, 0),
+	DMA_CHANNEL(0x00008100, 0x40, 0),
+	DMA_CHANNEL(0x00008180, 0x40, 0),
+	DMA_CHANNEL(0x00008200, 0x40, 0),
+	DMA_CHANNEL(0x00008280, 0x40, 0),
+	DMA_CHANNEL(0x00008300, 0x40, 0),
+	DMA_CHANNEL(0x00008380, 0x40, 0),
+	DMA_CHANNEL(0x00008400, 0x40, 0),
+	DMA_CHANNEL(0x00008480, 0x40, 0),
+	DMA_CHANNEL(0x00008500, 0x40, 0),
+	DMA_CHANNEL(0x00008580, 0x40, 0),
+	DMA_CHANNEL(0x00008600, 0x40, 0),
 };
 
 static const struct sh_audmapp_channel r8a7790_audmapp_channels[] = {
@@ -1393,21 +1393,21 @@ static const struct sh_dmadesc_slave_config r8a7790_sysdma_slaves[] = {
 };
 
 static const struct sh_dmadesc_channel r8a7790_sysdma_channels[] = {
-	DMA_CHANNEL(0x00008000, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008080, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008100, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008180, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008200, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008280, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008300, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008380, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008400, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008480, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008500, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008580, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008600, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008680, 0x40, 0x00000080),
-	DMA_CHANNEL(0x00008700, 0x40, 0x00000080),
+	DMA_CHANNEL(0x00008000, 0x40, 0),
+	DMA_CHANNEL(0x00008080, 0x40, 0),
+	DMA_CHANNEL(0x00008100, 0x40, 0),
+	DMA_CHANNEL(0x00008180, 0x40, 0),
+	DMA_CHANNEL(0x00008200, 0x40, 0),
+	DMA_CHANNEL(0x00008280, 0x40, 0),
+	DMA_CHANNEL(0x00008300, 0x40, 0),
+	DMA_CHANNEL(0x00008380, 0x40, 0),
+	DMA_CHANNEL(0x00008400, 0x40, 0),
+	DMA_CHANNEL(0x00008480, 0x40, 0),
+	DMA_CHANNEL(0x00008500, 0x40, 0),
+	DMA_CHANNEL(0x00008580, 0x40, 0),
+	DMA_CHANNEL(0x00008600, 0x40, 0),
+	DMA_CHANNEL(0x00008680, 0x40, 0),
+	DMA_CHANNEL(0x00008700, 0x40, 0),
 };
 
 static struct sh_dmadesc_pdata sysdma_platform_data = {
-- 
1.8.3.2

